srcpkgs/muse/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'muse'
+pkgname=muse
+version=4.0.0
+revision=1
+wrksrc="muse-${version/pre/-pre}"
```

Comment:
No longer necessary?

Reply:
Yeah, you right. 
Removed the substitution.